### ISSUE TITLE
[Feature Request] AppleScript and Shortcuts Interfaces. 

### ISSUE THREAD

**Issue by andrewcrook** [davidwernhart/AlDente#638]
expanding  on #637 

Also add AppleScript and Shortcuts interfaces to control and alter settings.

**Comment by ConnorBosman** [comment]
Expanding on this, what about maybe using the Mac URL shortcut to automatically trigger shortcuts? That would be great imo, for example by automatically triggering a shortcut upon a specific battery percentage.

https://support.apple.com/guide/shortcuts-mac/run-a-shortcut-from-a-url-apd624386f42/mac

**Comment by andrewcrook** [comment]
@ConnorBosman do you mean so you can add hooks that AlDente can call shortcut . 
So it can start shortcuts on certain events?

**Comment by ConnorBosman** [comment]
Exactly. The current tasks is nice, but it’s not very useful. I’d for example like to turn on a fan automatically when my battery gets too hot.
